bin/helm-build

```diff
@@ -2,8 +2,18 @@
 
 set -e
 
+restoreValues() {
```

Comment:
Maybe consolidate setting and restoring the values?

```suggestion
setValues() {
    sed -i "s/$1/$2/" charts/linkerd2/values.yaml
    sed -i "s/$1/$2/" charts/linkerd2-cni/values.yaml
}
```

Then, setting and restoring would just be:

```
# Set version in Values files
setValues "{version}" $fullVersion
```

```
# Restore version in Values files
setValues $fullVersion "{version}"
```

Reply:
Good suggestion, will do :+1: